pom.xml

```diff
@@ -1697,10 +1696,14 @@
                 <plugin>
                     <groupId>com.mycila</groupId>
                     <artifactId>license-maven-plugin</artifactId>
-                    <version>2.6</version>
+                    <version>2.8</version>
                     <configuration>
                         <skip>${skipVerifications}</skip>
-                        <header>${licensePath}</header>
+                        <header>com/mycila/maven/plugin/license/templates/CPAL.txt</header>
```

Comment:
I can bet it is the same file as I personally did the pull request to the mycila project. And of course I tested locally before commiting.
For EE it has to remain as it is, the license is not included (obviously) in the license checker distribution.